README.md

```diff
@@ -117,6 +117,19 @@ In order to access the admin features of the TestTrack server, you must create a
 > Admin.create!(email: "myemail@example.org", password: "[something secret]")
 ```
 
+### Enabling attachments
+Variants can be associated with metadata to describe their effects in human-readable terms. To enable variant screenshots, you'll need some additional configuration in environment variables.
+
+- To store uploads on the local file system, set `ATTACHMENT_STORAGE` to `local`, and:
+  - `LOCAL_UPLOAD_PATH` (optional, defaults to `:rails_root/public/system/:class/:attachment/:id_partition/:style/:filename`)
+- To store uploads in S3, set `ATTACHMENT_STORAGE` to `s3`, and:
+  - `S3_ACCESS_KEY_ID` (required)
+  - `S3_SECRET_ACCESS_KEY` (required)
```

Comment:
it looks like paperclip requires that credentials be passed and won't lean on aws-sdk figuring it out for itself (not that I can tell, anyway). but yeah, I can change these to be the same names as aws-sdk would expect anyway, so `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`